packages/component-library/components/Dropdown/README.mdx

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I think we should move these into `site/` and not use the magical README :)

Reply:
I think I added these because there's already some READMEs for some things in `component-library` and these files would hopefully soon be deleted anyway when the component gets removed.

Reply:
Won't it just mean the next person needs to move the file instead? 🤔